assets/app/views/_template.html

```diff
@@ -0,0 +1,7 @@
+<li class="template">
```

Comment:
im not sure this template is still being used, was it intended to be used on the catalog?  i added a new template under the views/catalog/ folder